pkg/operator/api/api.go

```diff
@@ -100,15 +97,19 @@ func (c *Cluster) makeDeployment(cluster *mon.ClusterInfo) *extensions.Deploymen
 	return deployment
 }
 
-func (c *Cluster) apiContainer(cluster *mon.ClusterInfo) v1.Container {
+func (c *Cluster) apiContainer() v1.Container {
 	// need a different prefix on the env vars
-	monSecretVar := k8smon.MonSecretEnvVar()
-	adminSecretVar := k8smon.AdminSecretEnvVar()
+	monSecretVar := opmon.MonSecretEnvVar()
+	adminSecretVar := opmon.AdminSecretEnvVar()
+	endpointVar := opmon.MonEndpointEnvVar()
+	clusterNameVar := opmon.ClusterNameEnvVar()
 	monSecretVar.Name = "ROOK_OPERATOR_MON_SECRET"
 	adminSecretVar.Name = "ROOK_OPERATOR_ADMIN_SECRET"
+	endpointVar.Name = "ROOK_OPERATOR_MON_ENDPOINTS"
```

Comment:
is the setting of the Name field on these env vars something that the `opmon.*EnvVar()` funcs can do? or are these Name field values specific to the caller?